connexion/apis/aiohttp_api.py

```diff
@@ -16,7 +16,17 @@
 from connexion.handlers import AuthErrorHandler
 from connexion.jsonifier import JSONEncoder, Jsonifier
 from connexion.lifecycle import ConnexionRequest, ConnexionResponse
-from connexion.utils import is_json_mimetype, yamldumper
+from connexion.problem import problem
+from connexion.utils import Jsonifier, is_json_mimetype, yamldumper
+from werkzeug.exceptions import HTTPException as werkzeug_HTTPException
+
+try:
+    import ujson as json
+    from functools import partial
+    json.dumps = partial(json.dumps, escape_forward_slashes=True)
+
+except ImportError:  # pragma: no cover
+    import json
```

Comment:
This `import ujson` / `import json` block should be removed.

With this PR, `json.dumps` calls are centralized in the Jsonifier so that datetimes can be handled consistently.
As such, this file actually does not use this json import anymore, so it should be removed.

Reply:
thanks @cognifloyd - great catch

Reply:
This PR removed this import block.
The merge with master re-introduced it.
Please remove this import block again.

Reply:
My bad. Does it look right to you now?

I was forced to use the editor built into github to resolve the merge (since it isn't my PR), so I'm missing a bunch context I would normally have in my local environment.

Reply:
That resolves my only concern :) Thanks